docs/integrations/api.md

```diff
@@ -0,0 +1,115 @@
+# **API**
+
+Here you can find various Lido APIs which you can integrate in your app or website:
+
+## Lido APR
+
+API provides Ethereum and Lido staking APR, which include:
+
+### **Last Lido APR for stETH**
+
+The latest staking APR value. For Lido V1, we collected APR values by periodically fetching [oracle report events](https://docs.lido.fi/contracts/lido-oracle#receiver-function-to-be-invoked-on-report-pushes). For the V2 version, the value is calculated based on [rebase events](https://github.com/lidofinance/lido-dao/blob/e9509d77f010fec76899e25ccde785c8de47bd42/contracts/0.4.24/Lido.sol#L232).
```

Comment:
Please change the commit id to to https://github.com/lidofinance/lido-dao/commit/e45c4d6fb8120fd29426b8d969c19d8a798ca974 instead `e9509d77f010fec76899e25ccde785c8de47bd42`

Reply:
Done ✅